### ISSUE TITLE
Undefined index: paymentType

### ISSUE THREAD

**Issue by chesio** [wirecard/woocommerce-wcp#44]
### Prerequisites

- [x] I have my [server set up correctly](https://guides.wirecard.at/shop_plugins:wcp:woocommerce:installation).
- [?] I am using a [supported version of WooCommerce](https://guides.wirecard.at/shop_plugins:wcp:woocommerce:start) - I don't know if version 3.5 of WooCommerce is officialy supported?

### Versions

**WooCommerce version:** 3.5.5
**PHP version:** 5.6
**WordPress version:** 4.9.10

### Issue

**Problem description:**

There is a following entry in `debug.log`:
```
PHP Notice:  Undefined index: paymentType in /[redacted]/wp-content/plugins/woocommerce-wirecard-checkout-page/class-woocommerce-wcp-gateway.php on line 393
```

In addition, there is a failed order from the same time as the entry in `debug.log`. Unfortunately, i don't have any additional information (WooCommerce logging is not active).

**Steps to reproduce:**

Would have to try to contact the customer to ask if there was anything special about the way she tried to make the order - there are several succesfull orders made in the last time (with the same WooCommerce and WordPress version). Anyway, perhaps you'll have an idea what could go wrong and can be done about it?

**Comment by serucee** [comment]
Hello @chesio ,

sorry for the late response. We will look into the described issue and inform you as soon as possible.

Kind regards,
Wirecard Shop Systems Team

**Comment by serucee** [comment]
The issue was resolved with the latest release (v1.3.11)
